docs/articles/documentation/reference/test-api/testcontroller/doubleclick.md

```diff
@@ -36,5 +35,13 @@ test('Invoke Image Options Dialog', async t => {
 
 > The `t.doubleClick` action will not invoke integrated browser actions such as text selection.
 > Use it to perform double clicks that are processed by webpage elements, not the browser.
```

Comment:
Do you need to mention "not the browser"?

Reply:
@dirk-pieterse, in this case, it is needed to highlight that there is more than one way to process double-clicks, as it may not be obvious without this remark.